packages/engine/src/lib/services/variable-service.ts

```diff
@@ -141,6 +153,12 @@ export class VariableService {
         if (Object.keys(innerValidation.errors).length > 0) {
           errors[key] = innerValidation.errors;
         }
+      } else if (type === PropertyType.DATE_TIME) {
+        const inferredDateTime = this.getISODateTime(clonedInput, key);
+        if (isNil(inferredDateTime) && property.required) {
+          errors[key] = `expected ISO string, but found value: ${value}`;
+        }
+        clonedInput[key] = inferredDateTime;
```

Comment:
Great PR, can you add please add tests to `variable-service.test.ts`

You can run it using `npx nx test engine`

Reply:
Its done